src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx

```diff
@@ -173,8 +174,10 @@ const Conversations: React.FC<ConversationsProps> = ({
   const mdBreakpoint = useMatchMedia('(max-width: 1000px)');
 
   useEffect(() => {
-    setIsSidebarOpen(!mdBreakpoint);
-  }, [mdBreakpoint, setIsSidebarOpen]);
+    if (sideBarOpenStatus !== SidebarOpenStatus.MANUAL_OPEN && sideBarOpenStatus !== SidebarOpenStatus.MANUAL_CLOSED) {
+      setSidebarOpenStatus(mdBreakpoint ? SidebarOpenStatus.CLOSED : SidebarOpenStatus.OPEN);
```

Comment:
Here we don't care if status was `SidebarOpenStatus.CLOSED` or `SidebarOpenStatus.OPEN`, so if we will use instead of those two one `SidebarOpenStatus.AUTO` to persist status and true/false for `openStatus`, we can write here
```
if (sideBarOpenStatus === SidebarOpenStatus.AUTO) {
      setSidebarOpenStatus(!mdBreakpoint);
}
```

Reply:
yep agreed. Actually a `useEffect` is not even needed here since there are no effects in there. 

The entire code should be something like

```
const mdBreaking = ...
const sideBarOpen = sidebarStatus === AUTO ? mdBreaking : sidebarStatus === OPEN;
```